packages/fast-components-styles-msft/src/accent-button/index.ts

```diff
@@ -11,6 +11,16 @@ import {
     neutralFocus,
     neutralFocusInnerAccent,
 } from "../utilities/color";
+import {
+    applyHighContrastAccentSelectedFillHover,
+    applyHighContrastAccentSelectedHover,
+    applyHighContrastAdjustSelectOutline,
+    applyHighContrastDisabledBorder,
+    applyHighContrastDisabledFill,
+    applyHighContrastDoubleFocus,
+    applyHighContrastHyperLink,
+    applyHighContrastSelectedColorFill,
```

Comment:
Do these need to be functions? It seems like they could just be objects

Reply:
updated and converted to objects